src/sdl_wrappers.h

```diff
@@ -5,17 +5,21 @@
 // IWYU pragma: begin_exports
 #if defined(_MSC_VER) && defined(USE_VCPKG)
 #   include <SDL2/SDL.h>
-#   include <SDL2/SDL_ttf.h>
 #   include <SDL2/SDL_image.h>
+#   include <SDL2/SDL_ttf.h>
 #else
 #   include <SDL.h>
-#   include <SDL_ttf.h>
 #   include <SDL_image.h>
+#   include <SDL_ttf.h>
 #endif
 // IWYU pragma: end_exports
 
 #include <memory>
 
+#include "SDL.h"
```

Comment:
```suggestion
```

Reply:
Good point.  These SDL headers give the most grief!  Will have to look into why they're not being handled correctly any more...